README.md

```diff
@@ -164,8 +164,8 @@ when you make a commit.
 
 ## Configuration
 
-terraform-docs can be configured with a yaml file. Default name of this file is
-`.terraform-docs.yml` and the order for lookig for it is:
+terraform-docs can be configured with a yaml file. The default name of this file is
+`.terraform-docs.yml` and the path order for locating for it is:
```

Comment:
I think "for" after "locating" should be dropped now.

Reply:
I agree. I think this suggestion/thread can now be resolved.